### PR TITLE
HURUmap Location Highlight

### LINKED ISSUE #770: @hurumap/core/LocationHighlight

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR create moves LocationHighlight to the HURUMap package.

Fixes #770 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots
image
image
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kelvinkipruto** [comment]
@CodeForAfrica/tech Any comments on this?

File: packages/hurumap-core/src/LocationHighlight/LocationHighlight.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { render } from "@commons-ui/testing-library";
+import * as React from "react";
```
Comment (koechkevin) [inline packages/hurumap-core/src/LocationHighlight/LocationHighlight.test.js:2]:
```suggestion
import React from "react";
```

**Review by koechkevin — APPROVED** [review]

File: packages/hurumap-core/src/LocationHighlight/LocationHighlight.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        sx={(theme) => ({
+          fontSize: theme.typography.pxToRem(10),
+          fontWeight: 300,
+          lineHeight: 24 / 10,
+          textTransform: "uppercase",
+        })}
+      >
+        {title}
+      </Typography>
+      <Typography variant="body2">{isLoading ? "…" : value}</Typography>
```
Comment (kilemensi) [inline packages/hurumap-core/src/LocationHighlight/LocationHighlight.js:28]: How does one customise `Typography` for `title` and `value`?

File: packages/hurumap-core/src/LocationHighlight/LocationHighlight.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from "@mui/material";
+import React from "react";
+
+const LocationHighlight = React.forwardRef(function LocationHighlight(
+  {
+    title,
+    value,
+    isLoading,
+    TitleTypographyProps,
+    ValueTypographyProps,
```
Comment (kilemensi) [inline packages/hurumap-core/src/LocationHighlight/LocationHighlight.js:10]: Sort these alphabetically (Caps first).

**Review by kilemensi — APPROVED** [review]

**Pull request merged**